### PR TITLE
Fix the typo

### PR DESCRIPTION
**Pull request by chetan-v**
In the reference of #2753 

convert from `.tar or .tar.gz or .zip`  to `.tar or .tar.gz`

### CODE REVIEW

File: main/webapp/modules/core/MOD-INF/controller.js [hunk 1]
```diff
@@ -236,6 +236,7 @@ function registerImporting() {
    *  Extension to format mappings
    */
   IM.registerExtension(".txt", "text/line-based");
+  IM.registerExtension(".json.txt", "text/json");
```
Comment (wetneb) [inline main/webapp/modules/core/MOD-INF/controller.js:239]: You have committed two unrelated changes in the same PR again :)

**Comment by tfmorris** [comment]
If you give your branches meaningful names, it'll be easier to keep track of them. For example, this one could be `2753-import-typo` or something similar to remind you what issue # it is associated with and what the topic is.

After you create the new branch, you might want to investigate the `git cherry-pick` command to copy one of your commits to it.

**Comment by thadguidry** [comment]
@chetan-v Also, if you are new to Git and still learning all the CLI commands, you might want to use a GUI to help you along as well.

[Gitkraken](https://www.gitkraken.com/git-client) can be used for free with open source, such as our OpenRefine repo.

**Comment by chetan-v** [comment]
Oh! I fix it very soon
